src/Raven.Server/Documents/Replication/OutgoingReplicationHandler.cs

```diff
@@ -70,6 +71,7 @@ public class OutgoingReplicationHandler : IDisposable
 
         private readonly ConcurrentQueue<OutgoingReplicationStatsAggregator> _lastReplicationStats = new ConcurrentQueue<OutgoingReplicationStatsAggregator>();
         private OutgoingReplicationStatsAggregator _lastStats;
+        public TcpConnectionInfo ConnectionInfo;
```

Comment:
Caching this is a bad idea, the TCP Connection details may change (the remote server may reboot and get a different port, for example, we need to re-get it each time.

Reply:
so we will fail and start again with getting the tcpinfo?

Reply:
Shouldn't this be private?